components/cyclebutton/stories/template.js

```diff
@@ -2,20 +2,54 @@ import { html } from 'lit-html';
 import { classMap } from 'lit-html/directives/class-map.js';
 // import { ifDefined } from 'lit-html/directives/if-defined.js';
 
+import { Template as Icon } from "@spectrum-css/icon/stories/template.js"
+
 import '../index.css';
-import '../skin.css';
+import "@spectrum-css/actionbutton/index.css"
+import "@spectrum-css/actionbutton/themes/spectrum.css"
+import "@spectrum-css/actionbutton/themes/express.css"
 
 export const Template = ({
   rootClass = "spectrum-CycleButton",
   customClasses = [],
   size = "m",
-  // ...globals
+  initialIcon = "Play",
+  selectedIcon = "Pause",
+  isSelected,
+  ...globals
 }) => {
+  
   return html`
-    <div class=${classMap({
+    <button class=${classMap({
```

Comment:
Can you update the markup here to leverage the Template function here:

https://github.com/adobe/spectrum-css/blob/b734251f0c2abbdc559e870d65fab7af2aa21d2d/components/actionbutton/stories/template.js#L1

Something like: 
```
ActionButton({
  ...globals,
  customClasses: [rootClass],
  isQuiet: true, ...
})
```

Reply:
I thought about this but I didn't know how to add the functionality for Cycle Button. ActionButton's story doesn't allow me to pass in children... I can pass in an icon but I don't have a way to pass in the second icon. Unless I'm missing something there?